Assets/Scripts/LevelEditor/Editor.cs

```diff
@@ -483,6 +509,13 @@ public string GetJson()
             return json;
         }
 
+        public string GetPropertiesJson()
+        {
+            string json = string.Empty;
+            json = JsonConvert.SerializeObject(PropController.instance.properties);
+            return json;
+        }
```

Comment:
This can be simplified to:
```suggestion
        public string GetPropertiesJson() =>
            JsonConvert.SerializeObject(PropController.instance.properties);
```
Since this refactors old code, should probably be a separate commit and/or PR